zerver/tests/fixtures/mattermost_fixtures/direct_channel/export.json

```diff
@@ -13,6 +13,7 @@
 {"type":"post","post":{"team":"gryffindor","channel":"gryffindor-common-room","user":"harry","message":"Looks like this channel is empty","create_at":1553166567370,"reactions":[{"user":"ron","create_at":1553166584976,"emoji_name":"rocket"}],"replies":null,"attachments":[{"path":"20210622/teams/noteam/channels/mcrm7xee5bnpzn7u9ktsd91dwy/users/knq189b88fdxbdkeeasdynia4o/smaa5epsnp89tgjszzue1691ao/this is a file"}]}}
 {"type":"direct_channel","direct_channel":{"members":["ron","harry"],"favorited_by":null,"header":""}}
 {"type":"direct_channel","direct_channel":{"members":["ron","harry", "ginny"],"favorited_by":null,"header":""}}
+{"type":"direct_channel","direct_channel":{"members":["harry","ron", "ginny"],"favorited_by":null,"header":""}}
```

Comment:
I don't know what was the actual pattern involved in the real data, but here I went with permuting the order of members on purpose - feels like that simulates some kind of more clever/realistic "duplicate dm group" situation than just copy-and-pasting the line above.

Reply:
Sounds reasonable. My guess is they might just appear as more exact duplicates, but not sure.